burger/index.html

```diff
@@ -0,0 +1,92 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <title>Burger Rectaurant</title>
+        <link rel="stylesheet" href="style/general.css">
+        <link rel="stylesheet" href="style/nav-header.css">
+        <link rel="stylesheet" href="style/home.css">
+        <link rel="stylesheet" href="style/my-theme.css">
+        <link rel="stylesheet" href="style/main-theme.css">
+        <link rel="icon" type="image/x-icon" href="img/logo-burger.png">
+    </head>
+    <body class="my-theme">
+        <nav>
+           <a href="index.html">Home</a>
+           <a href="gallery.html">Gallery</a>
+           <a href="contact-us.html">Contact Us</a>
+        </nav>
+        <header>
+            <img class="logo-burger" src="img/logo-burger.png" alt="logo">
+            <div>
+                <p class="adress">Velika Vasylkivska 52, Kyiv, Ukraine, 02000</p>
+            </div>
+            <div class="number-time">
+                <p class="number-time-number">+38 (095) 815-43-44</p>
+                <p class="number-time-time">we deliver from 10:00 to 22:00</p>
+            </div>
+            <div class="butter">
+                <a class="butter-order" href="#">Order</a>
```

Comment:
change href attribute here
you have already contact us page

Reply:
fixed all changes